raiden-cli/src/app.ts

```diff
@@ -0,0 +1,42 @@
+import { Server } from 'http';
+import { Logger } from 'loglevel';
+import express, { Request, Response, NextFunction, Errback } from 'express';
+import createError from 'http-errors';
+import apiV1 from './routes/api.v1';
+
+let server: Server;
+
+function notFoundHandler(_request: Request, _response: Response, next: NextFunction) {
+  next(createError(404, 'Undefined resource'));
+}
+
+function internalErrorHandler(
+  error: Errback,
+  _request: Request,
+  _response: Response,
+  next: NextFunction,
+) {
+  console.log(error); // TODO: proper error logging
+  next(createError(500, 'Internal Raiden node error'));
+}
+
+const app = express();
```

Comment:
Global/module level variables are how we did it so far, but I'd say for this improvement/rewriting, it could be worth it to have these instances in some context/object, instead of being global, which could allow multiple/independent clients to be run in a single Node instance, needed e.g. for testing (in the future). Some functions could need some context binding at some point, but this could be useful.
What do you think, @kelsos ?

Reply:
I also prefer scoping as much as possible and exposing the bare minimum to the global level.

Reply:
Okay, so what does that mean now. What would you expect the module to expose and how should these instances be stored/handled. I must admit that from this conversation I don't know how to improve.

Reply:
Already discussed. Just a headsup: global var

Reply:
I close this discussion due to a rework we have decided on. This will require a new review anyways.